AirCasting/CreateSessionViews/TurnOnBluetoothView.swift

```diff
@@ -29,6 +30,20 @@ struct TurnOnBluetoothView: View {
             continueButton
                 .buttonStyle(BlueButtonStyle())
         }
+        .background(
+            NavigationLink(
+                destination: PowerABView(creatingSessionFlowContinues: $creatingSessionFlowContinues, urlProvider: urlProvider),
+                isActive: $isPowerABLinkActive,
+                label: {
+                    EmptyView()
+                }
+            )
+        )
+        .onAppear(perform: {
+            if CBCentralManager.authorization != .allowedAlways {
+                _ = bluetoothManager.centralManager
```

Comment:
same here. But if we need to do it in more places I'd think of some less nasty/hacky way. Maybe it should just not be `lazy var`?

Reply:
I really don't have a strong opinion, so I will do as You decide 😄